movie/src/components/header/AddMovie/AddMovie.scss

```diff
@@ -0,0 +1,4 @@
+.add-movie-button {
+  background-color: rgb(184, 173, 173);
```

Comment:
You should always prefer using variables instead of magic values. What if designer will say to change this color to another one? Also in the prooject may be a themed variables what prefer to use. Please, create and include variables scss to main scss file. Please read the docs how to use variables in sass.

Reply:
added colors file

Reply:
Looks much better